File: core/src/org/rexellentgames/dungeon/debug/HelpCommand.java
package org.rexellentgames.dungeon.debug;

import org.rexellentgames.dungeon.UiLog;

public class HelpCommand extends ConsoleCommand {
	{
		name = "/help";
		shortName = "/h";
		description = "prints help";
	}

	@Override
	public void run(Console console, String[] args) {
		for (ConsoleCommand command : console.getCommands()) {
			UiLog.instance.print(command.getName() + " " + command.getDescription());
		}
	}
}

File: core/src/org/rexellentgames/dungeon/debug/HelpCommand.kt
package org.rexellentgames.dungeon.debug

import org.rexellentgames.dungeon.util.Log

class HelpCommand : ConsoleCommand("/help", "/h", "Prints help") {
    override fun run(console: Console, args: Array<String>) {
        for (command in console.commands) {
            Log.info(command.name + " " + command.description)
        }
    }
}